Makefile

```diff
@@ -151,7 +151,8 @@ build-image-local:
 		-t ${REGISTRY}/${CERTSUITE_IMAGE_NAME}:${IMAGE_TAG} \
 		-t ${REGISTRY_LOCAL}/${CERTSUITE_IMAGE_NAME_LEGACY}:${IMAGE_TAG} \
 		-t ${REGISTRY}/${CERTSUITE_IMAGE_NAME_LEGACY}:${IMAGE_TAG} \
-		-f Dockerfile .
+		-f Dockerfile . ;\
+	docker image prune -f
```

Comment:
This one is risky, it may wipe other images that might be in use, if you could determine what intermediate images can be wiped out that would be great

Reply:
I'll take out the `docker image prune -f` now and we can merge the rest of the changes.  It reduces that dangling layer size from 5GB to 1GB and if DCI hit any more storage problems we can address it in another PR.